components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -46,6 +49,26 @@ export default class ConfirmIntegration extends React.Component {
         }
     }
 
+    copyLink = (e) => {
+        const textarea = this.refs.textarea;
+        copyTextToClipboard(e, textarea.props.values.token);
```

Comment:
Reading the props of a ref is a bad practice that I think has been deprecated. If you make a component for this like I mentioned elsewhere, you could just have it read the token/url/id from that component's props instead of using a ref.
```javascript
class CopyText extends React.PureComponent {
    static propTypes = {
        value: PropTypes.string.isrequired,
    };

    copyText = (e) => {
        e.preventDefault();
        copyTextToClipboard(e, this.props.value);
    };

    render() {...}
}
```